Calificativ.js

```diff
@@ -15,3 +15,25 @@
      1: IS
 */
 
+let calificativ = function (parametru) {
+  if (parametru <= 4) {
+    return "IS";
+  }
+  if (parametru == 5 || parametru == 6) {
```

Comment:
Poți simplifica puțin condițiile din `if`, punând `parametru < 7`.

Pentru că a trecut de testul de la `if`-ul precedent, știm cu siguranță că nota nu poate fi mai mică decât 5. Deci ne interesează să fie mai mică decât 7 pentru a da calificativul "S".